vnext/src/RNTester/RNTesterList.windows.ts

```diff
@@ -139,6 +139,10 @@ const APIExamples: Array<IRNTesterExample> = [
     key: 'AppStateExample',
     module: require('react-native/RNTester/js/AppStateExample'),
   },
+  {
+    key: 'AlertExampleUwp',
+    module: require('./AlertExampleUwp'),
```

Comment:
Still dont think we should write a new one of these.  We should use the alert tester page thats already part of react-native/RNTester

Reply:
In order to do that alert.windows.ts would have to be moved into facebook's alert.js, which I plan on doing in a separate pull request. This tester page is only temporary until that happens